packages/shared/src/lib/core/wallet/actions/send/createEvmToStardustTransaction.ts

```diff
@@ -60,12 +60,19 @@ export async function createEvmToStardustTransaction(
             return
         }
 
-        const assetAllowance = buildAssetAllowance(iscChain, transferredAsset, storageDepositRequired)
-        const contract = iscChain?.getContract(ISC_SANDBOX_ABI, ISC_MAGIC_CONTRACT_ADDRESS)
+        const targetAddress = buildIscL1AddressFromBech32Address(recipientAddress)
+        const assets = buildIscAssets(iscChain, transferredAsset, storageDepositRequired)
+        const { sendMetadata, sendOptions } = buildUnwrapAssetParameters()
+
+        const magicContracts = new IscMagicContracts(iscChain as IscChain)
         const data =
-            (await contract?.methods
-                .send(targetAddress, assetAllowance, adjustMinimumStorageDeposit, sendMetadata, sendOptions)
-                .encodeABI()) ?? ''
+            (await magicContracts.sandbox.encode('send', {
```

Comment:
```suggestion
            (await magicContracts.sandbox.send({
```

Reply:
This wont work the send function just wraps the send method builder on the contract from web3.js here we are specifically asking it to encode that function to be used as the data in a transaction object we build